fxa-client/sdks/android/library/src/main/java/org/mozilla/fxaclient/internal/FxaClient.kt

```diff
@@ -42,12 +42,17 @@ internal interface FxaClient : Library {
         }
     }
 
-    fun fxa_get_release_config(clientId: String, redirectUri: String, e: Error.ByReference): RawConfig?
-    fun fxa_get_custom_config(contentBase: String, clientId: String, redirectUri: String, e: Error.ByReference): RawConfig?
+    fun fxa_new(
+        contentUrl: String,
+        clientId: String,
+        redirectUri: String,
+        e: Error.ByReference
+    ): RawFxAccount?
 
-    fun fxa_new(config: RawConfig, e: Error.ByReference): RawFxAccount?
     fun fxa_from_credentials(
```

Comment:
Ah, this binding probably shouldn't exist since it's not available without feature = browserid, which is never turned on for android.

Unless I'm mistaken?